charts/celestia-local/templates/deployment.yaml

```diff
@@ -85,6 +85,13 @@ spec:
               port: bridge-rest
             failureThreshold: 30
             periodSeconds: 10
+          resources:
+            requests:
+              cpu: 1
+              memory: 4Gi
+            limits:
+              cpu: 2
+              memory: 8Gi
```

Comment:
Contraining bridge node on dev cluster since it can be an expensive service to run.